frontend/src/routes.js

```diff
@@ -107,6 +107,18 @@ const NotebookCreateForm = Loadable(
   lazy(() => import('./modules/Notebooks/views/NotebookCreateForm'))
 );
 
+const MetadataFormList = Loadable(
+  lazy(() => import('./modules/Metadata_Forms/views/MetadataFormList'))
+);
+
+const MetadataFormView = Loadable(
+  lazy(() => import('./modules/Metadata_Forms/views/MetadataFormView'))
+);
+
+const MetadataFormCreate = Loadable(
+  lazy(() => import('./modules/Metadata_Forms/views/MetadataFormEdit'))
+);
```

Comment:
I see this is `MetadataFormCreate` but points to `MetadataFormEdit`

Do we need this route anyway since the create is a modal on top of `MetadataFormList - 'metadata-forms'` path?

Reply:
I will remove this `MetadataFormEdit`. In part 3 It's implemented without separate page

Reply:
[83a4021](https://github.com/data-dot-all/dataall/pull/1444/commits/83a402182ecefdc74e512790422db7e08970bb0e)